.changeset/curvy-beans-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
```

Comment:
I'm not familiar with how Core releases use the changesets, but shouldn't this be a `minor` as it's net new functionality?

Reply:
Updated